code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -18,7 +18,7 @@
 	exp_requirements = 720
 	exp_type = EXP_TYPE_CREW
 	exp_type_department = EXP_TYPE_SECURITY
-	alt_titles = list("Security Commander", "Security Chief")
+	alt_titles = list("Security Commander", "Security Chief, Chief Security Officer")
```

Comment:
That's Moja Manley (Security Chief, Chief Security Officer).

Reply:
Shit yeah lemme fix